gui/mozregui/log_report.py

```diff
@@ -1,16 +1,25 @@
 from PyQt4.QtCore import QObject, pyqtSlot as Slot, pyqtSignal as Signal
-from PyQt4.QtGui import QPlainTextEdit, QTextCursor, QColor, \
-    QTextCharFormat
+from PyQt4.QtGui import QPlainTextEdit, QTextCursor, QColor
+from PyQt4.QtGui import QTextCharFormat, QMenu, QAction
+from PyQt4.QtGui import QTextBlockUserData, QActionGroup
```

Comment:
I would prefer to use only one 'from PyQt4.QtGui' - if you don't like the backslashes (I understand;), you can use parenthesis, like:
```python
from PyQt4.QtGui import (QPlainTextEdit, QTextCursor, QColor
                                           QTextCharFormat, QMenu, QAction, ...)
```